### PR TITLE
Add Danish language

### PR DESCRIPTION
**Pull request by henriettemoe**

### CODE REVIEW

File: h5p-bildetema/language/da.json [hunk 1]
```diff
@@ -0,0 +1,382 @@
+{
+  "semantics": [
+    {
+      "label": "Oversættelser",
+      "fields": [
+        {
+          "label": "Sprogkode",
+          "description": "Sprogkoden skal skrives med to tegn (engelsk: en, norsk bokmål: nb)",
```
Comment (boyum) [inline h5p-bildetema/language/da.json:8]: Totally unnecessary change

```suggestion
          "description": "Sprogkoden skal skrives med to tegn (engelsk: en, dansk: da)",
```

File: h5p-bildetema/language/da.json [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+        {
+          "label": "Footer: privacy statement URL",
+          "default": "/privacy-statement"
+        },
+        {
+          "label": "Footer: accessibility statement tekst",
+          "default": "Accessibility statement"
+        },
+        {
+          "label": "Footer: Tilgjengelighetserklæring URL",
```
Comment (boyum) [inline h5p-bildetema/language/da.json:116]:
```suggestion
          "label": "Footer: Erklæring om tilgængelighed URL",
```

File: h5p-bildetema/language/da.json (hunk 1 above)
Comment (henriettemoe) [reply h5p-bildetema/language/da.json:8]: It's only the english descriptions that are shown in WordPress actually, so not that necessary no.

Skjermbilde 2022-11-29 kl  15 13 35

File: h5p-bildetema/language/da.json [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+        {
+          "label": "Footer: privacy statement tekst",
+          "default": "Privacy statement"
+        },
+        {
+          "label": "Footer: privacy statement URL",
+          "default": "/privacy-statement"
+        },
+        {
+          "label": "Footer: accessibility statement tekst",
```
Comment (boyum) [inline h5p-bildetema/language/da.json:112]:
```suggestion
          "label": "Footer: Erklæring om tilgængelighed tekst",
```

File: h5p-bildetema/language/da.json [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+        },
+        {
+          "label": "Vietnamesisk",
+          "default": "Vietnamesisk"
+        }
+      ]
+    },
+    {
+      "label": "Default languages",
+      "description": "There should be up to three default languages. These are defaults for the current site language. We recommend adding three of them",
```
[the remark this answers is not in the document]
Comment (henriettemoe) [reply h5p-bildetema/language/da.json]: Added

**Review by boyum — APPROVED** [review]
